packages/wonder-blocks-grid/components/grid.js

```diff
@@ -161,8 +162,10 @@ export default class Grid extends React.Component<
     }
 
     render() {
+        // eslint-disable-next-line no-unused-vars
+        const {ssrSize, size, ...otherProps} = this.props;
         return (
-            <GridContext size={this.state.size} spec={this.props.spec}>
+            <GridContext {...otherProps} size={this.state.size}>
                 {this.props.children}
             </GridContext>
```

Comment:
Won't `children` get passed `twice` to `GridContext`?  I usually pull out `children` as well when destructuring `this.props`.

Reply:
Hmm - perhaps? It doesn't seem to have an impact though. I can change it?